src/assets/i18n/fr.json5

```diff
@@ -2092,9 +2431,7 @@
   "item.edit.bitstreams.save-button": "Sauvegarder",
 
   // "item.edit.bitstreams.upload-button": "Upload",
-  "item.edit.bitstreams.upload-button": "Ajouter",
-
-
+  "item.edit.bitstreams.upload-button": "Télécharger",
```

Comment:
"Téléverser" may be better.

Reply:
@pilasou  Yes, of course "upload/download" would be better served by "téléversement/téléchargement" (and related verb "téléverser/télécharger"). I think that we are more familiar with these terms than elsewere in the French-speaking world where "télécharger" is used for both loading modes, i.e. upstream and downstream  (and the user must understand which one in context). That's what I would have put in a fr_CA translation, but I think I didn't dare to make this kind of translation in a more global use context. But if the community agrees, I will be happy to make this kind of change. These terms also have the advantage of being more precise.

Reply:
@mhvezina Ok, let's stick to "Télégargement".

Reply:
@pilasou You mean "Téléchargement" ;-)